OneCommunityGlobal/HighestGoodNetworkApp
# Nida Fix - Save Changes Fix on User Profile

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/be357ce2-ac29-4848-98ed-486c61965f70)


# Fixes (bug list priority medium)
- Added `setOriginalProjects(projects)` and ` setOriginalTeams(teams)` functions in each of the methods, `checkIsProjectsEqual()` and `checkIsTeamsEqual()` respectively.

## Main changes explained:
- Added `setOriginalProjects(projects)` and `setOriginalTeams(teams)` functions in each of the methods, `checkIsProjectsEqual()` and `checkIsTeamsEqual()` respectively, to ensure that the latest project value and the latest team value is being set when the save button is clicked.

## How to test:
1. check into current branch
2. log as admin user
3. go to User’s profile → Projects tab → Assign/Delete projects → Save changes
4. verify that when assigning or deleting projects in a user’s profile, the save changes warning does not appear, hence indicating that the changes have been saved on first click.

## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/72791192-c810-40fe-bfd6-b388e263dcfa
